kyuubi-server/src/main/scala/org/apache/kyuubi/engine/EngineRef.scala

```diff
@@ -96,32 +125,6 @@ private[kyuubi] class EngineRef(
     case _ => user
   }
 
-  @VisibleForTesting
-  private[kyuubi] val subdomain: String = conf.get(ENGINE_SHARE_LEVEL_SUBDOMAIN) match {
-    case Some(_subdomain) => _subdomain
-    case None if clientPoolSize > 0 =>
-      val poolSize = math.min(clientPoolSize, poolThreshold)
-      if (poolSize < clientPoolSize) {
-        warn(s"Request engine pool size($clientPoolSize) exceeds, fallback to " +
-          s"system threshold $poolThreshold")
-      }
-      val seqNum = enginePoolBalancePolicy match {
-        case "POLLING" =>
```

Comment:
Why remove the `POLLING` policy?

Reply:
ok  i will pull master code and merge it .